### ISSUE TITLE
Conda support

### ISSUE THREAD

**Issue by Casyfill** [qnzhou/PyMesh#110]
it would be nice to support conda installation

**Comment by qnzhou** [comment]
I agree.  I feel it shouldn't be hard, but I appreciate any help I can get.

**Comment by Walshaikhli** [comment]
It would be great to have conda install option. I have been trying to install Pymesh for 3 days now without luck so far.

**Comment by timeanddoctor** [comment]
anyone installed in conda success now?

**Comment by neoglez** [comment]
I made a PR to start with it:
https://github.com/PyMesh/PyMesh/pull/123

**Comment by dfsp-spirit** [comment]
@neoglez Where do I get that wheel file?

(I tried to install pymesh under Ubuntu 16.04 and 18.04, and it's a nightmare. An easier installation path would be a great feature, imo.)

**Comment by qnzhou** [comment]
@dfsp-spirit Sorry to hear that.  The wheel files are at https://github.com/PyMesh/PyMesh/releases

**Comment by dfsp-spirit** [comment]
Thanks! 

No worries, most likely it's just my lack of understanding how one should install python packages. ;) I'll get it to work sooner or later, I'm sure.

**Comment by tbarton16** [comment]
@neoglez does that install all the c++ dependencies as well?

**Comment by neoglez** [comment]
@tbarton16 It should, yes. Although i didn't test it all.

**Comment by geoffsmith** [comment]
If it's useful for anybody, I've created a conda package using conda-build: https://github.com/geoffsmith/pymesh-conda

If you're on linux (and I'll add OSX soon) you can do `conda install -c hotfigs pymesh` though I'm not sure how much I'll be able to maintain this... if this is useful, would probs be better to make an official build on conda-forge instead :)

**Comment by timkartar** [comment]
> conda install -c hotfigs pymesh

@geoffsmith you just saved 100s of hours of my life. Thank youuuu !

**Comment by wrk226** [comment]
> conda install -c hotfigs pymesh

You saved my life! Thank you sooooooo much!

**Comment by viviyanran** [comment]
> If it's useful for anybody, I've created a conda package using conda-build: https://github.com/geoffsmith/pymesh-conda
> 
> If you're on linux (and I'll add OSX soon) you can do `conda install -c hotfigs pymesh` though I'm not sure how much I'll be able to maintain this... if this is useful, would probs be better to make an official build on conda-forge instead :)

It is not working in Nov 13, 2023 when I  just try.
